apps/mobile/src/store/keychains/stacks/utils.ts

```diff
@@ -1,6 +1,16 @@
 import { entitySchema } from '@/store/utils';
+import {
+  AddressVersion,
+  createStacksPrivateKey,
+  getPublicKey,
+  publicKeyToAddress,
+} from '@stacks/transactions';
+import { deriveStxPrivateKey } from '@stacks/wallet-sdk';
 import z from 'zod';
 
+// FIXME fix this deprecated import
+import { mnemonicToRootNode } from '@leather.io/bitcoin';
```

Comment:
this is in crypto pkg

Reply:
I'll be removing this whole file in favour of the newer balance queries we have